pages/dashboard.tsx

```diff
@@ -41,6 +41,24 @@ export default function DashboardPage() {
   const twitterSubject = user?.twitter?.subject || null;
   const discordSubject = user?.discord?.subject || null;
 
+  const authenticateWithServer = async () => {
```

Comment:
nit: I would rename this `authorizeWithServer` just for consistency with how we frame it in docs